cli/compilers/ts.rs

```diff
@@ -290,22 +329,12 @@ impl TsCompiler {
     self: &Self,
```

Comment:
Oops - I see that the real "compile_async" is here.. ok ignore my previous comment.